src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -8657,6 +8657,51 @@ bool MergeTreeData::initializeDiskOnConfigChange(const std::set<String> & new_ad
     return true;
 }
 
+void MergeTreeData::loadPrimaryKeys()
+{
+    // Define the states of parts that need to be processed
```

Comment:
Nit: ClickHouse uses `///` instead of `//`.

Reply:
Fixed

Reply:
Still using `//` instead of `///`